src/encoder.c

```diff
@@ -262,6 +263,7 @@ static int _fmt_halffloat(nanocbor_encoder_t *enc, uint16_t half)
     return res;
 }
 
+#if FLT_MANT_DIG != DBL_MANT_DIG
```

Comment:
```suggestion
#if __SIZEOF_DOUBLE__ != __SIZEOF_FLOAT__
```

?

Reply:
Thanks I was actually looking for a macro like that one..